tests/backup/backup_stork_test.go

```diff
@@ -358,10 +358,15 @@ var _ = Describe("{BackupAndRestoreWithNonExistingAdminNamespaceAndUpdatedResume
 		log.FailOnError(err, "Unable to remove stork custom admin namespace")
 		err = DeleteAppNamespace(newAdminNamespace)
 		log.FailOnError(err, "Unable to delete custom admin namespace")
-		log.Infof("Deleting backup schedule policy")
-		for _, scheduleName := range scheduleNames {
-			err = DeleteSchedule(scheduleName, SourceClusterName, orgID, ctx)
-			dash.VerifySafely(err, nil, fmt.Sprintf("Verification of deleting backup schedule - %s", scheduleName))
+		log.Infof("Deleting backup schedule")
+		bkpScheduleEnumerateReq := &api.BackupScheduleEnumerateRequest{
+			OrgId: orgID,
+		}
+		allSchedules, err := Inst().Backup.EnumerateBackupSchedule(ctx, bkpScheduleEnumerateReq)
```

Comment:
Why are we deleting all the schedules? We are deleting the schedules that are not created as part of this test.